pkg/chunker/file/file.go

```diff
@@ -31,6 +32,13 @@ func Size(s int) Option {
 	}
 }
 
+// Follow file updates using inotify().
+func Follow() Option {
```

Comment:
nit: The convention for our functional opts has been to prepend `With` (`WithFollow`). Not a blocker for me.

Reply:
I can rename both options, as above there's `Size()` as well

Reply:
Ahh missed that one.